code/game/objects/items/fireaxe.dm

```diff
@@ -70,11 +70,27 @@
 	base_icon_state = "bone_axe"
 	name = "bone axe"
 	desc = "A large, vicious axe crafted out of several sharpened bone plates and crudely tied together. Made of monsters, by killing monsters, for killing monsters."
+///SKYRAT EDIT ADDITION BEGIN - Slight antimob axe buff
+///Bonus force dealt against certain factions. Blatantly stolen from the arrow code, because shitcode
+	var/faction_bonus_force = 9
+///Any mob with a faction that exists in this list will take bonus damage/effects
+	var/static/list/nemesis_factions = list("mining", "boss")
```

Comment:
I think we should avoid adding variables to core items. We should talk on discord about making it a component instead, which would make less copy-pasting code between items.

I do enjoy this idea though of dealing bonus damage against certain factions.